src/app/util/attribute-storage.h

```diff
@@ -208,6 +208,8 @@ CHIP_ERROR GetSemanticTagForEndpointAtIndex(chip::EndpointId endpoint, size_t in
 CHIP_ERROR SetTagList(chip::EndpointId endpoint,
                       chip::Span<const chip::app::Clusters::Descriptor::Structs::SemanticTagStruct::Type> tagList);
 
+CHIP_ERROR emberAfGetEndpointUniqueIdForEndPoint(chip::EndpointId endpoint, chip::MutableCharSpan & epUniqueIdMutSpan);
```

Comment:
Again, weird to have an unconditional declaration that will then lead to link errors if CHIP_CONFIG_USE_ENDPOINT_UNIQUE_ID is false....